ansible/egress_assess.yml

```diff
@@ -8,7 +8,8 @@
       ansible.builtin.include_role:
         name: assessment_tool
       vars:
-        assessment_tool_archive_src: https://github.com/tarrell13/Auto-Egress-Assess/tarball/main
+        # Later versions require Python 3.13 or later
```

Comment:
Can this comment please be expanded to mention that our current OS doesn't have Python 3.13?